tests/sqlparser_redshift.rs

```diff
@@ -281,3 +281,9 @@ fn test_sharp() {
         select.projection[0]
     );
 }
+
+#[test]
+fn test_create_view_late_binding() {
```

Comment:
It would be nice to have the names mirror the SQL as much as possible

```suggestion
fn test_create_view_no_schema_binding() {
```